ext/mysql2/client.c

```diff
@@ -563,23 +578,20 @@ static VALUE do_query(void *args) {
       rb_raise(cMysql2Error, "read_timeout must be a positive integer, you passed %ld", sec);
     }
     tvp->tv_usec = 0;
```

Comment:
While you're in here, would you move the tv_usec = 0 to be directly below the tv_sec = sec line?